test/core/QCheck2_expect_test.ml

```diff
@@ -182,7 +182,7 @@ module Shrink = struct
       (fun xs -> (*print_list xs;*) List.length xs < 4332)
 
   let list_equal_dupl =
-    Test.make ~name:"lists equal to duplication" ~print:length_printer
+    Test.make ~name:"lists equal to duplication" ~print:Print.(list int)
       Gen.(list_size size_gen small_int) (*Gen.(list small_int)*)
```

Comment:
Why keep some commented out code, like this, or like in `test/core/QCheck_expect_test.ml`?

Reply:
These were old combinations that I had considered.
Leaving them as comments was not ... ideal 😄
They have been removed in the latest commit.